api/service.py

```
@@ -170,11 +176,11 @@ def _check_credential_security():
         if metrics_user == weak_user and metrics_password == weak_pass:
             _USING_DEFAULT_CREDENTIALS = True
             print("\n" + "=" * 70)
-            print("🔴 CRITICAL SECURITY WARNING: Using default/weak credentials!")
+            print("[CRITICAL] SECURITY WARNING: Using default/weak credentials!")
             print("=" * 70)
             print(f"Detected credentials: {get_secret_masked('metrics_user')}/{get_secret_masked('metrics_password')}")
```

Ruff (0.14.10)

Undefined name `get_secret_masked`

(F821)